src/components/pages/ParentFamily/CreateNewStudent.js

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from 'react';
+import '../../../styles/ParentStyles/index.less';
+
+const CreateNewStudent = props => {
+  // const [checked, setChecked] = useState([]);
+
+  // const handleChange = (id, e) => {
+  //     e.preventDefault();
+  //     const copyChecked = [...checked];
+  //     const indexOfId = copyChecked.indexOf(id);
+  //     console.log('INDEXOFID', indexOfId);
+  //     if(indexOfId === -1){
+  //         copyChecked.push(id);
+  //     }else{
+  //         copyChecked.splice(indexOfId, 1);
+  //     }
+  //     console.log('COPYCHECKED', copyChecked);
+  //     setChecked(copyChecked);
+  // };
+
+  return (
+    <div className="create-new-student-modal">
+      <h2 className="create-new-student-title">Create New Student</h2>
+      <div className="exit" onClick={() => props.setModal(false)}>
+        X
+      </div>
```

Comment:
on line 25 instead of X we can use ant-design's icon 
I also did this and fixed it 
import { CloseOutlined } from '@ant-design/icons';
and instead of line 24~26 replace it with
<CloseOutlined  className="exit" onClick={() => props.setModal(false)}/>
might need to check that styling applied through className will take effect, since sometimes you might need to use !important to override the antdesign

Reply:
Thanks for this! I will try to implement this when speaking with Ryan Hamblin tomorrow.